packages/create-app/template-svelte/package.json

```diff
@@ -10,5 +10,6 @@
     "@sveltejs/vite-plugin-svelte": "^1.0.0-next.11",
     "svelte": "^3.37.0",
     "vite": "^2.3.7"
-  }
+  },
+  "type": "module"
```

Comment:
Maybe we could have it under the "version" field so it will be more clear to find out?

Reply:
done